dlfcn.go

```diff
@@ -33,6 +33,9 @@ func init() {
 // A second call to Dlopen with the same path will return the same handle, but the internal
 // reference count for the handle will be incremented. Therefore, all
 // Dlopen calls should be balanced with a Dlclose call.
+//
+// This function is not available on Windows.
+// Use [golang.org/x/sys/windows.LoadLibrary] behind a //go:build windows build tag.
```

Comment:
Maybe,

```
Use [golang.org/x/sys/windows.LoadLibrary] for Windows instead.
```

? It's because `//go:build` is not needed if the file has `_windows` suffix.

Reply:
Also, `NewLazyDLL` and `NewLazySystemDLL` are other options.

Reply:
Should I list all three options?

Reply:
I think so. (Offtopic: Personally `NewLazySystemDLL` is the best option, but the comment doesn't have to say this is the best)